### DISCUSSION TITLE
Antd5 的 css 最佳实践是什么？如何才能优雅地改写组件的样式。

### DISCUSSION THREAD

**Discussion by ByYogi** [ant-design/ant-design#44395] [Q&A] [answered]
对 `ConfigProvider` 还是有点难懂。我如果想实现响应式怎么实现，如果我只想对某个局部组件改一下样式，就要写一个`ConfigProvider`包裹吗？ 有些组件有多个dom，想要精确覆盖样式要怎么做。
对 `ConfigProvider`的印象是全局配置，而不是局部样式更改。
一般想要使用`ClassNames`来实现样式覆写。但是各种变量获取，定位dom啥的感觉好难搞

**Comment by MadCcc** [comment]
https://ant.design/docs/react/customize-theme-cn
参考这篇文档

**Comment by ByYogi** [comment, marked as answer]
参考这个知乎文章：https://zhuanlan.zhihu.com/p/636143897
这个库：https://ant-design.github.io/antd-style
